### ISSUE TITLE
How to use static library

### ISSUE THREAD

**Issue by fibnochi** [twelve17/openalpr-ios#1]
I have created openalpr-xcode.a static library following given instruct. I have also imported library into my project but when I try to include alp.h in my project I receive warning saying there is no such file.

**Comment by twelve17** [comment]
What do you have set for the header search paths?

**Comment by fibnochi** [comment]
Actually issue is static library was being built for device not for simulator and it was causing error. When I build my project for device it was working fine. 

I have resolved all the issue but do not know how to use it in my project as there is no documentation. If you can provide me some documentation about library usage or just tell me how to use it in mobile that will be grate help. Because when I run the project i get openalpr.conf file not found at /etc/openalpr/openalpr.conf.

**Comment by twelve17** [comment]
You need to import the openalpr.conf file as a resource in your project (do not copy as a reference, have it copy to your project). Once there, you can pass the Alpr constructor the path to the file.

**Comment by fibnochi** [comment]
Can you please tell me where is openalpr.conf file. I have copied and changed extension of openalpr.conf.in in config folder. Is this is the file I should use?

**Comment by fibnochi** [comment]
It will be nice if you can post complete way of calling alpr library. I mean some documentation will be nice.

**Comment by twelve17** [comment]
Nah, that is a template file.  You need to compile openalpr first, which will create the openalpr.conf file (without the .in), which you can then add to your project.

**Comment by twelve17** [comment]
I would like to at some point, but it takes a lot of work and I am still working on a project that I have to complete first.

**Comment by twelve17** [comment]
The openalpr project has a complete way to call the library. Have you taken a look at it?

**Comment by fibnochi** [comment]
Does not your code compile alpr and generate .conf file? If not then do tell me how to get .conf file.

I have checked and followed main.cpp code to call alpr library but it gives me error openalpr.conf missing.

**Comment by twelve17** [comment]
I compiled the code per the openalpr instructions.  I believe that created a openalpr.conf file in the $(installation prefix)/etc.  I then imported that file into my Xcode project.  Because the file is no longer in the default location, you will need to tell Alpr the new location, which is now a resource bundle in your app.  You will need to pass this path to the [alpr constructor](https://github.com/openalpr/openalpr/blob/master/src/openalpr/alpr.h#L114).

**Comment by fibnochi** [comment]
I have followed your instructions to compile but no .conf file is generated. If you can share your file that will be nice.

**Comment by twelve17** [comment]
I believe it's better for you to understand how to get the file yourself.  If you compile openalpr per their instructions, the file should exist at `src/config/openalpr.conf` after the `make` step.   I believe I used the steps in [the hard way for Linux, which work for Mac OS X too](https://github.com/openalpr/openalpr/wiki/Compilation-instructions-(Ubuntu-Linux)).  Can you try with that procedure?

**Comment by twelve17** [comment]
For v2 of this project, I have replaced the old instructions and bash scripts with a Ruby script to build the OpenALPR and dependency Frameworks, as well as the OpenALPR Xcode project.  A sample app is included.  These changes have been pushed to the master branch, and the old version has been moved to branch `1_0`.  Feel free to check out this new script and see if it works for you.  I will no longer be working on or supporting the `1_0` codebase.
